pipelines/_SingleFrame.yaml

```diff
@@ -3,6 +3,8 @@ tasks:
   isr: lsst.ip.isr.IsrTask
   characterizeImage: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
   calibrate: lsst.pipe.tasks.calibrate.CalibrateTask
+  writeSourceTable: lsst.pipe.tasks.postprocess.WriteSourceTableTask
+  transformSourceTable: lsst.pipe.tasks.postprocess.TransformSourceTableTask
```

Comment:
That makes sense to mirror what we had before. Can subsets refer to each other? e.g.
```
subsets:
  processCcd:
      - isr
      - characterizeImage
      - calibrate
  singleFrame:
    subset:
      - processCcd
      - writeSourceTable
      - transformSourceTable
    description: Single frame processing that includes table transformations
```

Reply:
Couldn't get the DECam repo going in time to add for all cameras, so @natelust take a look at https://github.com/lsst/obs_subaru/pull/341/files

Reply:
So currently this is the only place that labeled subsets cant be used as a substitute for labels. This is because at the time we didnt think it was worth adding code and complexity to track and handle cyclical definitions, to save a few lines in a (mostly) static file. If you think this is a good feature to have, we can add it.